test/contrib/automotive/can_xcp.uts

```diff
@@ -0,0 +1,316 @@
+% Regression tests for the XCP_CAN
```

Comment:
I refactored all my unit tests which use vcan_sockets, recently. I highly recommend to copy my initialisation code from there to have a identical setup, running on all travis machines.

Init-code: https://github.com/secdev/scapy/blob/ff43e450709605314c4739287e83c588a974a502/test/contrib/automotive/ccp.uts#L1-L96

Cleanup-code: https://github.com/secdev/scapy/blob/ff43e450709605314c4739287e83c588a974a502/test/contrib/automotive/ccp.uts#L1044-L1053

Usage example of sockets: https://github.com/secdev/scapy/blob/ff43e450709605314c4739287e83c588a974a502/test/contrib/automotive/ccp.uts#L959-L992